Content.Client/Power/APC/ApcVisualizerSystem.cs

```diff
@@ -13,6 +13,14 @@ protected override void OnAppearanceChange(EntityUid uid, ApcVisualsComponent co
         if (args.Sprite == null)
             return;
 
+
```

Comment:
Minor nit about the extraneous whitespace added. Not a big deal; I'll fix this in case you don't have time to look at this before I get around to testing this in-game.

Reply:
Thank you for the feedback, I removed the extra empty lines